### DISCUSSION TITLE
kube-proxy Grafana dashboard is empty.

### DISCUSSION THREAD

**Discussion by bip7686** [prometheus-operator/kube-prometheus#1196] [General] [answered]
I can't find a place where the kube-proxy is being queried, but the kubernetes-mixin project contains a grafana dashboard for the kube-proxy, which doesn't seem to work in my case.
Was it maybe forgotten to implement or remove the dashboard from the kubernetes-mixin project?

**Comment by paulfantom** [comment, marked as answer]
You need to enable kube-proxy to expose metrics on IP other that localhost and configure PodMonitor for it. We have a stale PR to make this work OOTB at https://github.com/prometheus-operator/kube-prometheus/pull/518, would you like to pick it up? :)

**Reply by bip7686** [reply]
Yes I'll take a look! :)
